ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -1035,3 +1036,26 @@ def test_funnel_step_breakdown_limit(self):
         # assert that we give 5 at a time at most and that those values are the most popular ones
         breakdown_vals = sorted([res[0]["breakdown"] for res in result])
         self.assertEqual(["5", "6", "7", "8", "9"], breakdown_vals)
+
+    def test_funnel_cohort_breakdown(self):
```

Comment:
Also, this test looks like a great candidate for `breakdown_cases.py` , so once the other PR goes in, please add it there instead :) - this ensures no surprises with the other orderings.